apps/system/style/zindex.css

```diff
@@ -334,3 +335,14 @@
 #screen > [data-z-index-level="accessibility-captions"] {
   z-index: 2147483647;
 }
+
+/**
+ * Global overlays are on top of everything, all the time.
+ */
+#screen > [data-z-index-level="global-overlays"] {
+  z-index: 2147483647;
+}
+
+#screen > [data-z-index-level="global-overlays"] > .appWindow.overlayWindow {
+  z-index: 2147483647;
```

Comment:
Be careful - that means you have chance to miss device menu and cannot shutdown/reboot your phone when there is an overlay window